src/blocks/block-author-profile/components/edit.js

```diff
@@ -158,11 +158,11 @@ export default class Edit extends Component {
 						) }
 						keepPlaceholderOnFocus
 						value={ profileContent }
-						formattingControls={ [
-							'bold',
-							'italic',
-							'strikethrough',
-							'link',
+						allowedFormats={ [
+							'core/bold',
+							'core/italic',
+							'core/strikethrough',
+							'core/link',
```

Comment:
It looks like the `core/` namespace is [now required](https://github.com/WordPress/gutenberg/blob/498951c04da14e817463ec76eed543048ce1de97/packages/block-editor/src/components/block-navigation/editor.js#L21-L26).